app/forms/instance-create.tsx

```diff
@@ -95,11 +106,24 @@ const baseDefaultValues: InstanceCreateInput = {
   userData: null,
 }
 
+const useBootDiskItems = (projectSelector: PP.Project) => {
+  const { data: disks } = useApiQuery('diskList', {
+    query: { ...projectSelector, limit: 1000 },
```

Comment:
Tested this with 500 disks in the response and it doesn't seem to get bogged down at all.